recipes/stir/meta.yaml

```diff
@@ -0,0 +1,52 @@
+
+{% set name = "STIR" %}
+{% set version = "v3_0_1" %}
+{% set sha256 = "4f257fa7a4b15c9d6eddf981d80b79fc27a3ed10df4b8184d507d82a22f3596e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/olivierverdier/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [py2k]
```

Comment:
If this package doesn't support windows, I think this should be `skip: true  # [py2k or win]`

Reply:
Good point. It should support windows (I will remove the libgcc requirement), but I'm not sure how to test that.